experimental/traffic-portal/src/app/core/profiles/profile-detail/profile-detail.component.html

```diff
@@ -0,0 +1,61 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card>
+    <tp-loading *ngIf="loading"></tp-loading>
```

Comment:
This is missing title buttons that provides the following (added links to the TPv1 requests if you wanna see how we do it in angularjs):
* View Servers
* Clone [Profile](https://github.com/apache/trafficcontrol/blob/43e0af1f9e50f8e63639c4e83f5a8bb1aabcc7b0/traffic_portal/app/src/common/api/ProfileService.js#L102)
* Export [Profile](https://github.com/apache/trafficcontrol/blob/43e0af1f9e50f8e63639c4e83f5a8bb1aabcc7b0/traffic_portal/app/src/common/api/ProfileService.js#L116)
* Queue Updates by [Profile](https://github.com/apache/trafficcontrol/blob/43e0af1f9e50f8e63639c4e83f5a8bb1aabcc7b0/traffic_portal/app/src/common/api/ProfileService.js#L141)
* Clear Updates by [Profile](https://github.com/apache/trafficcontrol/blob/43e0af1f9e50f8e63639c4e83f5a8bb1aabcc7b0/traffic_portal/app/src/common/api/ProfileService.js#L154)

Reply:
yes, thought of adding those while we are implementing those features. May be i will add and keep it disabled.